csvcubed/csvcubed/utils/validators/schema.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+import jsonschema
+from jsonschema.exceptions import ValidationError
+
+
+def validate_dict_against_schema(value: dict, schema: dict, log: logging.RootLogger) -> list[ValidationError]:
```

Comment:
Again, I think the logger should be owned by this module, not passed in.

Reply:
log created at module level, no longer passed.